internal/pkg/externalplugins/tars/tars.go

```diff
@@ -249,19 +248,17 @@ func getRefBranch(ref string) string {
 func HandleAll(log *logrus.Entry, ghc githubClient, config *plugins.Configuration,
 	externalConfig *tiexternalplugins.Configuration) error {
 	log.Info("Checking all PRs.")
-	orgs, repos := config.EnabledReposForExternalPlugin(PluginName)
-	if len(orgs) == 0 && len(repos) == 0 {
+	_, repos := config.EnabledReposForExternalPlugin(PluginName)
+	if len(repos) == 0 {
 		log.Warnf("No repos have been configured for the %s plugin", PluginName)
 		return nil
 	}
 
-	var queries []string
-	for _, org := range orgs {
-		queries = append(queries, searchQueryPrefix+` org:"`+org+`"`)
-	}
-
 	if len(repos) > 0 {
```

Comment:
Is it possible to reduce the nesting level, for example, if `len(repos) <= 0` directly return.

Reply:
Good idea.